paper/paper.md

```diff
@@ -0,0 +1,198 @@
+  - name: Bernd Bischl
+    orcid: 0000-0001-6002-6980
+    affiliation: "2, 3"
+affiliations:
+ - name: TU Dortmund University, Germany
+   index: 1
+ - name: Department of Statistics, LMU Munich, Germany
+   index: 2
+ - name: Munich Center for Machine Learning (MCML), Germany
+   index: 3
+ - name: Department of Cancer Genetics, Institute for Cancer Research, Oslo University Hospital, Norway
+   index: 4
+ - name: Leibniz Institute for Prevention Research and Epidemiology (BIPS), Bremen, Germany
+   index: 5
+ - name: Faculty of Mathematics and Computer Science, University of Bremen, Germany
+   index: 6
+ - name: Certilytics, Inc., 9200 Shelbyville Rd, Louisville, KY, 40222, USA
+   index: 7
+ - name: Friedrich-Alexander-Universität Erlangen-Nürnberg (FAU), Erlangen, Germany
+   index: 8
+ - name: Department of Statistics and Finance/International Institute of Finance, School of Management, University of Science and Technology of China, Hefei, Anhui, China
+   index: 9
+ - name: School of Environmental and Forest Sciences, University of Washington, Seattle
+   index: 10
+ - name: Department of Geography, LMU Munich, Germany
+   index: 11
+ - name: Statistical Consulting Unit StaBLab, LMU Munich, Germany
+   index: 12
+ - name: Wake Forest University School of Medicine, Department of Biostatistics and Data Science, Division of Public Health SciencesWinston-Salem, North Carolina
+   index: 13
+ - name: University of North Carolina Wilmington
+   index: 14
+ - name: OSPO Now
+   index: 15
+ - name: multi, 8 passage Brûlon, 75012 PARIS, France
+   index: 16
+ - name: ServiceTitan, Inc., Glendale, California
+   index: 17
+date: XXX December 2024
+bibliography: paper.bib
+---
+
+# Summary
+
+The [`mlr3extralearners`](https://mlr3extralearners.mlr-org.com/) [`R`](https://www.r-project.org/) [@R] package is a community-driven package that integrates external machine learning algorithms into the [`mlr3`](https://mlr3.mlr-org.com/) [@Lang2019] ecosystem.
+The `mlr3` ecosystem is a versatile toolbox for machine learning in `R` and is targeted towards both practitioners and researchers [@Bischl2024].
+At its core, the `mlr3extralearners` package provides a standardized interface for machine learning and connects many R packages implementing machine learning algorithms into a unified framework.
+The package currently wraps **85 different learning algorithms** from many different R packages, for tasks such as classification, regression, and survival analysis.
+This enables users to seamlessly access and utilize these learners directly within their workflows.
+It also facilitates large-scale empirical benchmark experiments, leveraging the `mlr3` framework's parallelization and optimization capabilities [@benchlargescale] as well as enabling efficient hyperparameter tuning [@bischl2023].
+An overview of all `mlr3` learners, including those introduced through `mlr3extralearners`, is available on the [mlr3 website](https://mlr-org.com/learners.html).
```

Comment:
consider making this a reference or footnote, not a link. When I render the pdf, the link is not visible as such.